products/opensuse/main.pm

```diff
@@ -344,7 +344,7 @@ sub load_reboot_tests() {
     if (check_var('ARCH', 's390x')) {
         loadtest "installation/reconnect_s390";
     }
-    if (installyaststep_is_applicable()) {
+    if (installyaststep_is_applicable() && !check_var('ARCH', 's390x')) {
```

Comment:
first one is true, but how to extract it to installyaststep_is_applicable and explicitly excluding this one test but not the others?